content/hackathons/usw24/index.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: "Unikraft Summer Workshop 2024"
+description: Unikraft Summer Workshop 2024, July 1 - July 20, 2024
+startDate: 2024-07-01
+registerForm: https://forms.gle/6AMFoCoqJqKGdBgX6
+image: TODO
+---
+
+## Unikraft Summer Workshop 2024
+
+Unikraft Summer Workshop 2024 (*USW'24*) is a **free and virtual** unikernel and library operating systems workshop held by members of the Unikraft community.
+
+The three-week event holds a number of starter tutorials, workshops and lessons on how to get build Unikraft unikernels for production systems.
+
+The workshops will be hands-on and will take place for six days, between July 1 and July 12, 2024, 4pm-7pm CEST, in English, virtually via [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
+Topics include building unikernels, benchmarking, debugging, porting applications, virtualization and platform specifics.
+The workshops will be followed by a week of working on the final project.
+We will have support sessions online to help with the project, and on Saturday, 20 July 2024, we will have the final hackathon, online, on the [Unikraft's Discord server](), and for those who can make it, live, at [University POLITEHNICA of Bucharest]().
```

Comment:
```suggestion
We will have support sessions online to help with the project.
On Saturday, 20 July 2024, 9am-5pm CEST, we will have the final hackathon.
The hackathon will take place in hybrid format, in person, at the [The National University of Science and Technology POLITEHNICA of Bucharest](https://upb.ro/en/), and, online, [Unikraft's Discord server](https://bit.ly/UnikraftDiscord).
```

Reply:
Participants will receive a participation diploma.
The first three teams will get special prizes.
All hackathon in-person participants will get a Unikraft T-shirt.

Reply:
> special prizes

Do we know / can we announce the top prizes? It might rise more attention.